### DISCUSSION TITLE
Where is 1.0.0-preview3?

### DISCUSSION THREAD

**Discussion by yoshiask** [microsoft/WindowsAppSDK#1641] [Q&A] [answered]
Howdy!

After hearing the news that WinAppSDK + WinUI 3 will be the primary way to build Windows apps going forward, I starting porting an existing UWP app of mine to WinUI 3 Desktop. (This UWP app had barely any reason to be UWP, since it circumvented the sandbox and requires a custom installer.) However, the lastest version of WinAppSDK that's supported by all the libraries I use (namely, [WCT](https://github.com/CommunityToolkit/WindowsCommunityToolkit) and [Microsoft.Xaml.Behaviors.WinUI.Managed](https://www.nuget.org/packages/Microsoft.Xaml.Behaviors.WinUI.Managed/2.0.4)) is Project Reunion 0.8.

Is there an ETA for 1.0.0-preview3? Being stuck on 0.8 is quite problematic, mostly because I can't tell if the issues I'm running into were already fixed or not.

Thank you.

**Comment by yoshiask** [comment, marked as answer]
1.0.0-preview3 was released two hours ago :)
https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.0.0-preview3

**Reply by shelllet** [reply]
Are there any example of how to Activate a Window in unpackaged app?

**Reply by BenJKuhn** [reply]
@shelllet, great question. Can you start it as a separate discussion item so it's not lost / buried?
